tox.ini

```diff
@@ -75,8 +77,8 @@ extras = test
 commands =
     {[testenv]commands}
 deps =
+    ansible>=1.10.0a2,<2.11
```

Comment:
This is clearly a typo :D

Reply:
oh. oops. I'll wait for the CI to finish and fix this annoying typo.